ajax/libs/cheval/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "cheval",
+  "filename": "cheval.min.js",
+  "description": "Copy to the clipboard using JavaScript without writing JS.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ryanpcmcquen/cheval"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com:ryanpcmcquen/cheval.git",
+    "fileMap": [{
+      "basePath": "",
+      "files": [
+        "*.js"
+      ]
+    }]
+
+  },
+  "keywords": [
+    "Clipboard",
+    "Copy",
+    "Copy text",
+    "Copy to clipboard"
+  ],
+  "author": "Ryan P.C. McQuen <ryanpcmcquen@member.fsf.org> (https://ryanpcmcquen.org/)",
+  "homepage": "https://ryanpcmcquen.org/cheval",
+  "license": "GPL-2.0+"
```

Comment:
Let's modify to `"GPL-2.0"` here.
Because GPL-2.0 is abbreviation for GNU General Public License v2.0
There is no GPL-2.0`+` license.
Many thanks.

Reply:
Is this following the SPDX license syntax? If so, isn't `GPL-2.0+` the only way to denote `GPLv2` or any later version? Page 82 of the SPDX 2.0 spec says to use a `+` character to represent that. If I put `GPL-2.0` that would mean version `2.0` _only_.

Reply:
Yes, we follow SPDX license syntax. This is our [license-list](https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json)
And `GPL-2.0+` has been deprecated, as this link show https://spdx.org/licenses/GPL-2.0+.html 
Or, maybe we can use [`"deprecated_GPL-2.0+"`](https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json#L113).
What do you think?
Many thanks.

Reply:
It does say deprecated, but then it points to the _license expression syntax_ at the bottom of the page here:
https://spdx.org/licenses/

Which leads you to page 96 of the current spec:
https://spdx.org/sites/cpstandard/files/pages/files/spdxversion2.1.pdf

Wherein they confirm that the unary operator `+` means that version or any later version:

>An SPDX License List Short Form Identifier​ with a unary"+" operator suffix to represent the current version of the license or any later version.  For example: GPL­2.0+

Maybe I am reading it wrong, but it seems that the `+` is still valid. What do you make of it?

Reply:
The short identifier is `GPL-­2.0+` on SPDX license website.
But the short identifier license list on CDNJS is `deprecated_GPL-2.0+`.
So, you are right.
@PeterDaveHello Would you mind joining discuss the license issue with us?
I thought the short identifier of `GPL-2.0+` on CDNJS was changed to `deprecated_GPL-2.0+` because [the license has been deprecated since 2.0rc2](https://spdx.org/licenses/GPL-2.0+.html).
But I don't know why CDNJS's license list doesn't follow the short identifier of SPDX.
The author want this library under GPLv2 or any later version, and use `GPL-2.0+` which is showed on SPDX is a good way to denote. But if use `deprecated_GPL-2.0+` of CDNJS's license list, I think this is a little bit bad news for the author.

Many thanks.

Reply:
help ping @PeterDaveHello

Reply:
Any movement on this? @PeterDaveHello?

Reply:
@sufuf3, I switched to the MPL-2.0 license, can we merge now?